File: plus-minus.js
function plusMinus(arr) {
    let [positive, negative, zero] = [0, 0, 0]
    
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] > 0) {
            positive++
        } else if (arr[i] < 0) {
            negative++
        } else if (arr[i] === 0) {
            zero++
        }
    }
    
    console.log((positive / arr.length).toFixed(6))
    console.log((negative / arr.length).toFixed(6))
    console.log((zero / arr.length).toFixed(6))

}

File: plus-minus.py
def plusMinus(arr):
    positive = 0
    negative = 0
    zero = 0
    
    for i in range(len(arr)):
        if int(arr[i]) > 0:
            positive += 1
        elif int(arr[i]) < 0:
            negative += 1
        elif int(arr[i]) == 0:
            zero += 1
    
    print(f'{(positive / len(arr)):.6f}')
    print(f'{(negative / len(arr)):.6f}')
    print(f'{(zero / len(arr)):.6f}')
